From 7d044dfd08509c7d2f8fee3cfa364d52d7b5fcfc Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Fri, 23 Dec 2011 10:51:55 +0800
Subject: [PATCH 0789/2463] ENGR00170922 [MX6Q]fix build error if disable SMP

fail log:

cpu_regulator-mx6.c: In function 'mx6_cpu_regulator_init':
cpu_regulator-mx6.c:87: error: 'loops_per_jiffy' undeclared

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 arch/arm/mach-mx6/cpu_regulator-mx6.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_regulator-mx6.c b/arch/arm/mach-mx6/cpu_regulator-mx6.c
index f973c5a..779ed62 100644
--- a/arch/arm/mach-mx6/cpu_regulator-mx6.c
+++ b/arch/arm/mach-mx6/cpu_regulator-mx6.c
@@ -32,6 +32,8 @@ static int cpu_op_nr;
 static struct cpu_op *cpu_op_tbl;
 extern struct cpu_op *(*get_cpu_op)(int *op);
 
+extern unsigned long loops_per_jiffy;
+
 static inline unsigned long mx6_cpu_jiffies(unsigned long old, u_int div,
 					      u_int mult)
 {
-- 
1.7.7.4

